.claude/skills/code-review/SKILL.md

```
@@ -0,0 +1,50 @@
+### 3. UI 품질
+- ❌ `ComponentProps` 확장 없이 네이티브 props를 중복 정의한다 → ✅ `ComponentProps<'el'>` 확장
+- ❌ `cn()` 없이 클래스를 문자열로 이어 붙인다 → ✅ `cn()` 사용
+- ❌ 인터랙티브 요소에 라벨/`aria-*`·시맨틱 태그가 없다 → ✅ 접근성 속성 부여
+
+### 4. 성능 (vercel-react-best-practices 스킬)
+- ❌ 워터폴(불필요한 순차 await)이 있다 → ✅ 병렬화
+- ❌ 불필요한 리렌더 유발(인라인 객체 prop, effect 남용)이 있다 → ✅ 안정 참조/파생
+- ❌ 무거운 컴포넌트에 `next/dynamic`이 없다 → ✅ 동적 임포트
+
+## 결과 형식
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)